docs/ja/ai-integration/tools-json.md

```
@@ -281,15 +271,13 @@ Claude Desktop の設定ファイルを編集します:
 }
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)